radar/api/serializers/patients.py

```diff
@@ -22,17 +22,17 @@
 
 class RecruitedDateField(fields.DateField):
     def get_attribute(self, patient):
-        return patient.recruited_date()
+        return patient.recruited_date(group_type="HOSPITAL")
 
 
 class RecruitedGroupField(TinyGroupSerializer):
     def get_attribute(self, patient):
-        return patient.recruited_group()
+        return patient.recruited_group(group_type="HOSPITAL")
 
 
 class RecruitedUserField(TinyUserSerializer):
     def get_attribute(self, patient):
-        return patient.recruited_user()
+        return patient.recruited_user(group_type="HOSPITAL")
```

Comment:
not all users are from hospitals or form part of a group. Check your own user ID in group users and I think you will find you are not there. Could cause an error